ccdproc/image_collection.py

```diff
@@ -652,10 +663,17 @@ def _generator(self, return_type,
 
             file_name = path.basename(full_path)
 
+            if 'unit' not in ccd_kwargs:
```

Comment:
I agree, thanks for pointing that out. However, it seems that I still need to set a default unit, otherwise the reader is going to throw this error: ``ValueError: a unit for CCDData must be specified.``